src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackAssessment/FeedbackAssessment.tsx

```diff
@@ -65,18 +67,29 @@ export const FeedbackAssessment = ({
 
       const answer = await getEncryptedAnswers(answersToEncrypt);
 
-      if (!appletId || !answerId) return;
+      if (!appletId) return;
 
       setItemIds(updatedItemIds);
 
-      await createAssessmentApi({
-        appletId,
-        answerId,
-        answer,
-        itemIds: updatedItemIds || [],
-        reviewerPublicKey: accountId,
-        assessmentVersionId: getAssessmentVersion(isLastVersion, assessmentVersions),
-      });
+      if (answerId) {
+        await createAssessmentApi({
+          appletId,
```

Comment:
Could you please move props to a variable for reuse in both cases? Only `answerId/submitId` is unique.